docs/Quickstart.mdx

```diff
@@ -98,6 +101,25 @@ Now type the following to install Enarx:
 
 ![linux install enarx](/img/install/linux_install_enarx.png)
 
+If you are on the `X86_64` host, it is recommended to also install the signature file:
+```sh
+wget https://github.com/enarx/enarx/releases/download/v0.6.4/enarx-x86_64-unknown-linux-musl.sig
+sudo install -D -m 444 -o root -g root enarx-x86_64-unknown-linux-musl.sig /usr/lib/enarx/enarx.sig
+```
+
+# RPM Installation
```

Comment:
Package manager installation sections should come first and the binary install should be last